### PR TITLE
Nav bar private pages dev

### LINKED ISSUE #66: As a developer, npm install eslint dependency in front and backend code

### PR DESCRIPTION
**Pull request by XeniyaDob**
## Description

NavBar routes are fixed from Upper case to lower case letters. Logo added. Logout functionality added. The search option is temporarily deleted.

## Related Issue
closes #66 

## Acceptance Criteria
AC:

Logout should delete the data from local storage (Token and users) and redirect the user to '/'

When clicking on MyKitchenApp on NavBar, direct them to '/dashboard'

All the links in the navBar should direct the user to their respective pages, 
The search page has been deleted for now. Search functionality will be added later as another ticket.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    ✓| :bug: Bug fix              |
|   ✓| :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before
Screen Shot 2022-11-15 at 10 11 41 AM

### After

Screen Shot 2022-11-21 at 9 36 23 AM

### CODE REVIEW

File: src/components/Navbar/ResponsiveAppBar.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ import Tooltip from "@mui/material/Tooltip";
 import MenuItem from "@mui/material/MenuItem";
 import { NavLink } from "react-router-dom";
 
-const pages = ["Dashboard", "Search", "Favorite"];
-const settings = ["Profile", "History", "Logout"];
+const pages = ["dashboard", "search", "favorite"];
+const settings = ["profile", "history"];
```
Comment (nuriyealp) [inline src/components/Navbar/ResponsiveAppBar.js:17]: These two options, profile history, look lowercase. Should we make them an upper case or capital?
Screen Shot 2022-11-15 at 8 13 53 PM

Comment (XeniyaDob) [reply src/components/Navbar/ResponsiveAppBar.js:17]: Yes, they were in upper case, then they appear in upper case in the URL path also.

File: src/App.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ function App() {
               <Route path="profile" element={<Profile />} />
               <Route path="history" element={<History />} />
               <Route path="dashboard" element={<Dashboard />} />
-              <Route path="searchresult" element={<SearchResult />} />
+              <Route path="search" element={<SearchResult />} />
```
Comment (elenamagay) [inline src/App.js:34]: We need fix this later. The 'SEARCH" link on the navbar should open the search form, not navigate to the search result page

**Review by elenamagay — APPROVED** [review]

File: src/components/Navbar/ResponsiveAppBar.js (hunk 1 above)
Comment (CodergirlVS) [reply src/components/Navbar/ResponsiveAppBar.js:17]: There is no history route. We should not keep things if they are not yet ready for the users. Also I dont see the logout in the menu. So I may convert this PR to draft till your done working on all of it. Also where it says MyKitchen, is it staying like that or its going to be your logo there?

Comment (CodergirlVS) [reply src/components/Navbar/ResponsiveAppBar.js:17]: for display you can have it Capitalized but in the link it will pass the value which is lowercase

**Review by nuriyealp — APPROVED** [review]
Great job!

**Review by nuriyealp — APPROVED** [review]
Great job!

**Pull request merged**